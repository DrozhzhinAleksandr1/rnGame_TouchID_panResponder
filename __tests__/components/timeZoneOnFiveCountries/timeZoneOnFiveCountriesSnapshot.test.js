import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { localization } from '../../../app/common/localization';
import { TimeZoneOnFiveCountries } from '../../../app/components/indicatorScreen/timeZoneOnFiveCountries/timeZoneOnFiveCountries';

it('create snapshot TimeZoneOnFiveCountries', () => {
    let date = {};
    date.getHours = () => { return 10 };
    date.getMinutes = () => { return 10 };
    date.getSeconds = () => { return 10 };
    date.getTimezoneOffset = () => { return 240 };

    const renderer = new ShallowRenderer()
    let timeZoneOnFiveCountries = renderer.render(<TimeZoneOnFiveCountries date={date} />);
    expect(timeZoneOnFiveCountries).toMatchSnapshot();
});