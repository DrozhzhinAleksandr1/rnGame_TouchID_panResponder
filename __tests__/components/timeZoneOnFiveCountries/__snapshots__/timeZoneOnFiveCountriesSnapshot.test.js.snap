// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot TimeZoneOnFiveCountries 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#1D2E3E",
      "flex": 2,
      "marginLeft": 10,
      "marginRight": 10,
      "marginTop": 5,
      "shadowColor": "rgba(0, 0, 0, 0.15)",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 2,
      "width": 730,
      "zIndex": 1,
    }
  }
>
  <TimeZoneOnFiveCountriesGreed
    dataGreed={
      Array [
        Array [
          12,
          9,
          "New York",
          "-04",
        ],
        Array [
          7,
          9,
          "London",
          "+01",
        ],
        Array [
          23,
          9,
          "Tokyo",
          "+09",
        ],
        Array [
          22,
          9,
          "Sydney",
          "+10",
        ],
      ]
    }
    hour={14}
  />
  <Component
    style={
      Object {
        "height": "81%",
        "width": "100%",
      }
    }
  >
    <Component
      pointerEvents="none"
      style={
        Array [
          Object {
            "backgroundColor": "rgba(255, 255, 255, 0.75)",
            "height": "100%",
            "position": "absolute",
            "top": 0,
            "width": 2,
          },
          Object {
            "left": "59.039351851851855%",
          },
        ]
      }
    />
  </Component>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "minHeight": 14,
        "width": "100%",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "rgba(255, 255, 255, 0.75)",
            "fontSize": 11,
            "lineHeight": 14,
            "minHeight": 14,
            "paddingLeft": 2,
            "width": "100%",
          },
          Object {
            "paddingLeft": "59.039351851851855%",
          },
        ]
      }
    >
      14
      :
      10
       GMT
    </Text>
  </Component>
</Component>
`;
